resources/views/layouts/master.blade.php

```diff
@@ -1,8 +1,7 @@
 <!DOCTYPE html>
 <html>
-
 	<head>
-		<title>Lara | @yield('title', 'VedSt Default Title')</title>
+		<title>Lara | @yield('title', 'VedSt')</title>
```

Comment:
Why not remove the 'VedSt' entirely? None of the users know what it means

Reply:
+1